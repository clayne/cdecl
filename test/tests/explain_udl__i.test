cdecl @ @ -xc++ @ explain int operator"" _x() @ 65
