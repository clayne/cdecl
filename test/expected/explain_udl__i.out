explain int operator"" _x()
                          ^
27: syntax error: ")": parameter list expected
